cmd/juju/space/show.go

```diff
@@ -34,7 +34,24 @@ type ShowSpaceCommand struct {
 const ShowSpaceCommandDoc = `
 Displays extended information about a given space. 
 Output includes the space subnets, applications with bindings to the space,
-and a count of machines connected to the space.`
+and a count of machines connected to the space.
+
+Examples:
+
+Show a space by name:
+
+	juju show-space alpha
+
+Show a space by ID:
+
+	juju show-space 0
```

Comment:
I understand that this is part of the spec but I am wondering why we opted to expose (internal) space IDs to the operator when we we explicitly avoid doing so for `juju deploy --bind` and `juju bind` commands (client works with space names and the translation happens server-side).

Unless this is meant to use the **provider** space ID (for MAAS)?

Perhaps @mitechie can shed some light on this.